### ISSUE TITLE
[$20][MSFT-229] [Challenge Feed][NVD][Common] Screen reader read-out when user focus on a empty spaces

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2887]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges
2. Hover over the empty location marked in the screenshot

**Expected Result**
Should not read-out when user point to empty locations

**Actual Result**
Screen reader read-out when user focus on empty spaces

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F66

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096024 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096024 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096024 has been updated - it has been assigned to gets0ul.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by veshu** [comment]
@sushilshinde @anonymousjaggu @gets0ul Since this looks merged marking the label as ready for QA.

Revert me if I am wrong. Thanks!

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096758 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096758 has been updated - it has been assigned to gets0ul.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
Operating System: Windows
Browser: Chrome
Pass URL :https://test-community-app.topcoder-dev.com/challenges

**Comment by drasticdpk** [comment]
Passed on Windows  Chrome but fail on Edge browser, here is the reference  FYI @lakshmiathreya @veshu
https://monosnap.com/file/aGMOa3p6ogBWrpr8F6yMBJsqTlzDdf

**Comment by veshu** [comment]
@drasticdpk Is this passed on test environment with edge on windows?

**Comment by drasticdpk** [comment]
@veshu yes it is working on test environment with edge. For the same i was uploaded reference video too.
https://monosnap.com/file/aGMOa3p6ogBWrpr8F6yMBJsqTlzDdf

**Comment by drasticdpk** [comment]
Verified on Edge too.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096758<br/><br/>```This is an automated message for crazyk via Topcoder X```
